tools/src/launcher/Main.hs

```diff
@@ -365,9 +363,40 @@ main =
         pretty @Integer $ fromIntegral $ convertUnit @_ @Second ts
 
 
-generateTlsCertificates :: ConfigurationOptions -> FilePath -> FilePath -> M ()
-generateTlsCertificates ConfigurationOptions{..} tlsPath executable =
-    unlessM (liftIO $ doesDirectoryExist tlsPath) $ do
+data TlsPaths = TlsPaths
+    { tlsCaPath     :: FilePath
+    , tlsServerPath :: FilePath
+    , tlsKeyPath    :: FilePath
+    } deriving (Show)
+
+
+findTlsArgs :: [Text] -> M TlsPaths
+findTlsArgs args = do
+    tlsCaPath     <- findArg "--tlsca"   (pure . toString) args
```

Comment:
This will add one more runtime failure point  if someone changes how tls arguments are parsed from `NodeOptions`. Maybe exporting these constants "tlsca" "tlscert" and "tlskey" from `NodeOptions` module ?

Reply:
Not sure if it worth the effort here :/ 

- The failure will be quite clear
- There's no reason to change option names and whoever would want to that will probably be very careful in doing so
- We are planning on ditching completely the launcher moving forward